shared/chat/conversation/conversation.css

```diff
@@ -1,7 +1,6 @@
 .WrapperMessage-hoverBox {
   padding-bottom: 3px;
   padding-top: 3px;
-  padding-left: 56px;
```

Comment:
the hierarchy is different now. we have a strict left/right/top/bottom area now which makes this saner